tests/modules/cooler/merge/main.nf

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { COOLER_MERGE } from '../../../../modules/cooler/merge/main.nf' addParams( options: [:] )
+
+workflow test_cooler_merge {
+
+    input = [ [ id:'test' ], // meta map
+              [ file(params.test_data['homo_sapiens']['cooler']['test_merge_cool'], checkIfExists: true) ] ]
```

Comment:
Doesn't the "merge" file take multiple input files?

Reply:
Yes. I updated the test file to 2 input files.